### ISSUE TITLE
Use <<DataType>> for TextureAssociation

### ISSUE THREAD

**Issue by clausnagel** [opengeospatial/CityGML-3.0CM#89]
TextureAssociation should not be an object but a data type.

**Comment by clausnagel** [comment]
Ok, the current `<<ObjectType>>` stereotype seems to be correct. If we want to be able to reuse texture parameterizations (i.e. `TexCoordList` or `TexCoordGen`) per XLink like in CityGML 2.0, then `TextureAssociation` requires a `gml:id` and thus must be an object type.

In CityGML 2.0, `TexCoordList` and `TexCoordGen` had the `gml:id` - simply because there was no explicit `TextureAssociation` element as in CityGML 3.0. But it makes more sense to reference `TextureAssociation` because it also contains the `uri` target. Thus, `TexCoordList` and `TexCoordGen` can remain data types in CityGML 3.0.

In summary, this issue can be ignored. However, we should check whether [this example dataset](http://schemas.opengis.net/citygml/examples/2.0/appearance/Appearance-EPSG25832.gml) for CityGML 2.0 appearances with XLinks can still be expressed.
